README_DEVELOPERS.md

```diff
@@ -43,3 +43,25 @@ npm run test -- -- -t react
 ## Replace etalon
 
 To replace etalon image just remove it from `packages/devextreme-cli/testing/__tests__/__image_snapshots__` folder and run tests again
```

Comment:
```suggestion
To replace ab etalon image just remove it from `packages/devextreme-cli/testing/__tests__/__image_snapshots__` folder and run tests again
```

Reply:
This sentence will be fixed in another PR.